conf/locale/ar/LC_MESSAGES/djangojs.po

```diff
@@ -9684,6 +9723,45 @@ msgstr ""
 "فوق زر الخيار ، ثم احفظها باسم) {link_end} ثم {upload_link_start} حمّلها "
 "{link_end} إلى backpack."
 
+#: themes/edx-platform/adg/lms/templates/student_account/form_field.underscore
+msgid "Forgot password?"
+msgstr "استعادة كلمة المرور الخاصة بي"
+
+#: themes/edx-platform/adg/lms/templates/student_account/login.underscore
+#: themes/edx-platform/adg/lms/templates/student_account/register.underscore
+msgid "OR CONTINUE WITH"
+msgstr ""
+
+#: themes/edx-platform/adg/lms/templates/student_account/login.underscore
+msgid "New to the Omnipreneurship Academy?"
+msgstr ""
+
+#: themes/edx-platform/adg/lms/templates/student_account/login.underscore
+msgid "Create account"
+msgstr "إنشاء حساب"
+
+#: themes/edx-platform/adg/lms/templates/student_account/password_reset.underscore
+msgid ""
+"Please enter the email address you registered with. We will send you a "
+"password reset email."
+msgstr ""
+
+#: themes/edx-platform/adg/lms/templates/student_account/password_reset.underscore
+msgid "SEND"
+msgstr ""
+
+#: themes/edx-platform/adg/lms/templates/student_account/register.underscore
+msgid "Already have an account?"
+msgstr "هل لديك حساب {platformName} مسبقاً؟"
```

Comment:
@harris-irfan Please remove this `{platformName}` from the translated string. I think the translator has done this or I don't know where it came from.

Reply:
As discussed, we will be going to master with the current file, without incorporating the suggested change.